tests/components/google_assistant/__init__.py

```diff
@@ -230,4 +230,36 @@
     'traits': ['action.devices.traits.TemperatureSetting'],
     'type': 'action.devices.types.THERMOSTAT',
     'willReportState': False
+}, {
+    'id': 'lock.front_door',
+    'name': {
+        'name': 'Front Door'
+    },
+    'traits': ['action.devices.traits.LockUnlock'],
+    'type': 'action.devices.types.LOCK',
+    'willReportState': False
+}, {
+    'id': 'lock.kitchen_door',
+    'name': {
+        'name': 'Kitchen Door'
+    },
+    'traits': ['action.devices.traits.LockUnlock'],
+    'type': 'action.devices.types.LOCK',
+    'willReportState': False
+}, {
+    'id': 'lock.openable_lock',
+    'name': {
+        'name': 'Openable Lock'
+    },
+    'traits': ['action.devices.traits.LockUnlock'],
+    'type': 'action.devices.types.LOCK',
+    'willReportState': False
+}, {
+    'id': 'group.all_locks',
```

Comment:
We should not expose this group. That's dangerous. I'm fine with hardcoding `group.all_locks` to never be exposed.

Reply:
Okay! If I understand you correctly, we only want to hide it from cloud services, right? If so, I think https://github.com/home-assistant/home-assistant/pull/18233/commits/3595d83efeba3f9c805130512e45a964cd5979eb takes care of that.

Reply:
Perfect.